### DISCUSSION TITLE
How do I get notification when my issued was closed?

### DISCUSSION THREAD

**Discussion by qiulang** [community/community#113109] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

When my issued is closed I don't get email notification. I research it but I can't find a way to get notification. Currently I have check to **Recent activity** at my github login page regularly to see if my issue was disappeared or not. If it was disappeared I know it was closed.

I can also check [notification tab](https://github.com/notifications) but when I am watching several projects I may get many messages so it is not easy to see my issue. 

Is there any way, or any setting I miss, to let me get the notification for that ?

**Comment by nakiviar** [comment]
Hi @qiulang ,

Go to GitHub Settings:

Select "Settings" from the dropdown menu.
Navigate to Notification Settings: in the left sidebar, scroll down and click on "Notifications".
Under "Participating", ensure that "Issues" is checked.
Scroll down to the "Notification delivery" section. ensure that "Email" is selected under "Participating".

![image](https://github.com/community/community/assets/54564415/0d2fca89-662e-41a9-bc56-faa610190d87)

Let me know if it worked for you 🥸

**Comment by qiulang** [comment]
Thanks!

I indeed didn't turn on email. Now I just need to open an issue to see if that works.

<img width="799" alt="notification" src="https://github.com/community/community/assets/516243/ae59625b-ba3f-4015-b457-8181966b20c5">

**Comment by qiulang** [comment]
@nakiviar today I got an email notification about an old issue I opened before https://github.com/microsoft/vscode-copilot-release/issues/703.

So you had correctly answered my question and I don't know who & why someone labeled you comment as "This comment was marked as off-topic."  So I can not make it as "Mark as answer" Sorry
